worldedit-core/src/main/java/com/sk89q/worldedit/command/RegionCommands.java

```diff
@@ -112,8 +112,8 @@ public int set(Actor actor, EditSession editSession,
 
     @Command(
         name = "/line",
-        desc = "Draws a line segment between cuboid selection corners",
```

Comment:
`segment` is correct here, not sure why it was removed? Probably should be "Draws line segment*s*" now though.

Reply:
You're right, should've put more care into editing the strings. Coming right up.

Reply:
Do you believe this would be good?

> "Draws line segments between cuboid selection corners or convex polyhedral selection vertices"

It kinda irks me that it talks about segments in the plural but the cuboid selection will only generate one segment, i'm not a native English speaker though.

Reply:
That sounds fine to me...if you really want to be explicit about the separation, `Draws a line segment between cuboid corners, or line segments between convex poly vertices.` but that's getting verbose. Language-wise, saying `segments` is fine though.

Reply:
I see, thanks for your input. I will use what I came up with then.